packages/ckeditor5-image/src/imageresize/imageresizehandles.js

```diff
@@ -56,6 +56,11 @@ export default class ImageResizeHandles extends Plugin {
 		editingView.addObserver( ImageLoadObserver );
 
 		this.listenTo( editingView.document, 'imageLoaded', ( evt, domEvent ) => {
+			// The resizer must not be attached to images inside HTML embed preview.
+			if ( domEvent.target.matches( 'div.raw-html-embed__preview img' ) ) {
```

Comment:
The fix will work for html embed preview, but only for that.

On the horizon there's an inline image feature so the check should be specific for the CKEditor's image plugin.

In other words it should check if the target image is the one that we need to attach to not if it is invalid one.

TL;DR: the check should be something like `figure.image img` (to check).

The inline image would be just an `img` without a figure, hence the check.

Reply:
It's a good idea to check more than just one unsupported case (the HTML embed image previews) by reversing the logic and _allow_ only known (supported) images to be resized 👍🏻